api/mraa/common.h

```diff
@@ -234,6 +234,42 @@ unsigned int mraa_get_platform_pin_count(uint8_t platform_offset);
 char* mraa_get_pin_name(int pin);
 
 /**
+* Get pin number, board must be initialised.
```

Comment:
Are these methods returning pin numbers or indexes?

Perhaps:
Get GPIO index by name, board must be initialised